src/components/addComments.vue

```diff
@@ -0,0 +1,175 @@
+<template>
+  <section className='add_message_section'>
+    <form @submit.prevent='addMessageHandler' className='message_form'>
+      <textarea
+        className='message_textarea'
+        :value='currentTextAreaValue'
+        @input='updateInput'
```

Comment:
```suggestion
        v-model='currentTextAreaValue'
```
vue knows how to handle 2-way data binding. read about [v-model in docs](https://vuejs.org/guide/essentials/forms.html)